### DISCUSSION TITLE
OSD display options for multi-monitor systems

### DISCUSSION THREAD

**Discussion by cfhsncb** [linuxmint/discussions#272] [Ideas]
Currently the OSD pop-up displays on all monitors on multi-display systems. An option to choose between displaying on all monitors or primary monitor only (as set in "displays") would be a nice improvement.

**Comment by alkaris2** [comment]
OSD needs Styling Options, because the default look isn't always the best look to match desktop themes.

Right now the OSD style in Cinnamon 6.4 is very bland and doesn't suit my current desktop theme, so having different styling options would be nice to have. Give us the following tweaking options;

- OSD Color Tweaks
- OSD Look and Feel (font, font color, border, padding)
- OSD Positioning
- OSD Toggle option of displaying `<Device Name>` + Display `Volume %`, or just Display `Volume %`

in the simplest form of being able to tweak it should be direct CSS modification of OSD, ie; `font-size: 18px`, `padding: 20px`, `background: #3e3a39`, `color:#5294e2`

**Comment by JosephMcc** [comment]
You can already theme it with css. Your theme just needs to support it. Here are the relevant selectors in the default cinnamon theme: https://github.com/linuxmint/cinnamon/blob/master/data/theme/cinnamon-sass/widgets/_osd.scss#L11
